css/css-lists/parsing/list-style-shorthand.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Lists: list-style sets longhands</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists-3/#propdef-list-style">
+<meta name="assert" content="list-style supports the full grammar '<'list-style-position'> || <'list-style-image'> || <'list-style-type'>'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/shorthand-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_shorthand_value('list-style', 'square url("https://example.com/") inside', {
+  'list-style-position': 'inside',
+  'list-style-image': 'url("https://example.com/")',
```

Comment:
Mind using a non-remote URL? Maybe a data URI?

I might be being over-paranoid, but I've had to change some of these tests so they don't crash on Firefox's automation (we crash as soon as Firefox tries to hit the network on automation).

Reply:
Changed to use {{host}}